Makefile

```diff
@@ -87,7 +87,7 @@ harvest: deps
 
 	@# Build the harvest poller
 	@echo "Building poller"
-	@GOOS=$(GOOS) GOARCH=$(GOARCH) go build -trimpath -o bin/poller -ldflags=$(LD_FLAGS) cmd/poller/poller.go
+	@cd cmd/poller; GOOS=$(GOOS) GOARCH=$(GOARCH) go build -trimpath -o ../../bin/poller -ldflags=$(LD_FLAGS)
```

Comment:
this isn't needed is it?

Reply:
It's needed, As now we have 2 files in cmd/poller to compile: poller.go and asup.go.

Reply:
this breaks `go run cmd/poller/poller.go -p foo` let's move `asup.go` to `cmd/tools/asup` and then revert the change at line 90

Reply:
Done